<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_ConnexionServ">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>631</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Connexion au serveur</string>
   </property>
   <widget class="QLabel" name="label_IPserv">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse IP du serveur :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_numPort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Num√©ro de port du serveur :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ip">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>172.18.58.83</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_numPort">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>1025</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButto_Quitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>107</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Connexion">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>108</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_EtatConnexion">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>61</y>
      <width>241</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_EtatConnexion">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>41</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat de la connexion</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_InfoClients">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>631</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Informations Clients</string>
   </property>
   <widget class="QPushButton" name="pushButton_NomOrdi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom de l'ordinateur distant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_NomUser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom de l'utilisateur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Processeur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>OS de l'ordinateur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Architecture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Architecture processeur</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nomOrdi">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>421</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Architecture">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>421</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nomUsers">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>421</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Processeur">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>421</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButto_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>ClientWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
